content/docs/sidebar.json

```diff
@@ -94,13 +94,18 @@
     "slug": "user-guide",
     "source": "user-guide/index.md",
     "children": [
+      {
+        "label": "Glossary",
+        "slug": "glossary",
+        "source": "basic-concepts"
+      },
```

Comment:
needs to be moved further down the sidebar, question is where? CC @iterative/docs

Reply:
Agreed, that we should move it down. Probably even to the bottom of the UG?

Reply:
I think Glossary can be the last section.

Reply:
I moved glossary to the bottom of user-guide for now.